### DISCUSSION TITLE
Hide network requests in command log (question from Patterns webinar)

### DISCUSSION THREAD

**Discussion by jprealini** [cypress-io/cypress-realworld-app#734] [Q&A] [answered]
Hi.. this is the question I asked during the Patterns and Practices webinar
This is my scenario (it's simple). I need to seed my db, and for a given reason, the endpoint I use allows me to seed one item at a time... So, if I need to seed 50 items, I do 50 requests... I issue the seed command in my before() hook... so, at the beginning of my command log I get 50 request logs... I tried using the {log: false} here and there, but no luck...
Any way I can achieve this?

**Comment by amirrustam** [comment]
Can you share the specific code snippet of your before hook?

**Comment by jprealini** [comment]
Hi @amirrustam ... sure! 

This is my custom command from where I seed the db using and endpoint created just for that

```
Cypress.Commands.add('seedCollection', (seedFile) => {
    cy.fixture('../seeding/' + seedFile).then((fixture) => {
        var lista = new Array();
        var offset;
        fixture.forEach((transaction, index) => {            
            cy.request({
                method: 'POST',
                url: apiBaseUrl + 'transaction',
                body: transaction,
                headers: {
                    'Authorization': 'Bearer ' + Cypress.env('Token'),
                    'Content-Type': 'application/json;charset=UTF-8'
                }
            })
        })
    })
})
```

And this is my spec file's hooks section

```
describe('Dashboard', () => {
    before(() => {
        cy.userLogin()
        cy.cleanCollection('transactions')
        cy.seedCollection('transactionsForSmallCharts.json')
    })
...
})
```

And this is what I see in the command log (I would like to hide all those "seeding requests")

![image](https://user-images.githubusercontent.com/7246755/95375863-9dd53d80-08b6-11eb-832d-fc043fe54688.png)

Thanks!!!

**Comment by jprealini** [comment]
Hi @amirrustam ... just checking if you guys have come across any more people asking for something like this and if there is any way to achieve it (not that I desperately need it, just curious.. in fact, the project I was working one while making this question is finished... no hurries)

Thanks!

**Comment by amirrustam** [comment, marked as answer]
@jprealini only individual commands can have their logging disabled. Network requests will currently always display within the command log.
